functional-tests.sh

```diff
@@ -574,6 +576,204 @@ function test_config_host_add_error()
     log_success "$start_time" "${FUNCNAME[0]}"
 }
 
+function test_put_object_with_sse()
+{
+    show "${FUNCNAME[0]}"
+    start_time=$(get_time)
+    object_name="mc-test-object-$RANDOM"
+    cli_flag="${SERVER_ALIAS}/${BUCKET_NAME}=32byteslongsecretkeymustbegiven1"
+    #put encrypted object; then delete with correct secret key
```

Comment:
A comment should follow the bash style of `#<space><my_comment>`

Reply:
# done